modules/idlgen/plugin/src/sbt-test/sbt-mu-idlgen/srcProtoGenFromDirs/test

```diff
@@ -0,0 +1,3 @@
+> compile
```

Comment:
I changed it by `compile` since it's being generated in that `srcGen` is compiled in that stage